src/temp/diveButton/DiveIntoMap.module.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../theme/theme';
+.diveIntoMapMainContainer {
+  width: 306px;
+  height: 341px;
```

Comment:
Why is the width hardcoded? Shouldn't this take the full screen width (accounting for padding/margin outside the container)?

Height can be left as is, but we probably need to adjust it while combining this element.